### PR TITLE
Makes burning items actually hot

### PR DESCRIPTION
**Pull request by ChungusGamer666**
## About The Pull Request

Simply put, unless overriden, items with the ON_FIRE flag will always be at least 150º C (491,15 kelvin).
I didn't want to pack this change with https://github.com/tgstation/tgstation/pull/74803 because I feel this might have some weird unintended consequences, maybe.

## Why It's Good For The Game

Emergent gameplay like lighting a cigarette with a burning piece of paper, I guess.

## Changelog

:cl:
add: Burning items are now actually considered to be at a minimum, 150 degrees celsius by the game.
/:cl:

### CODE REVIEW

File: code/game/objects/items.dm [hunk 1]
```diff
@@ -879,7 +879,9 @@
 
 ///Returns the temperature of src. If you want to know if an item is hot use this proc.
 /obj/item/proc/get_temperature()
-	return heat
+	. = heat
+	if(resistance_flags & ON_FIRE)
+		. = max(., FIRE_MINIMUM_TEMPERATURE_TO_EXIST)
```
Comment (Mothblocks) [inline code/game/objects/items.dm:884]:
```suggestion
	if(resistance_flags & ON_FIRE)
		return max(heat, FIRE_MINIMUM_TEMPERATURE_TO_EXIST)
	return heat
```

No trivial use of `.` -- https://github.com/tgstation/tgstation/blob/master/.github/guides/STANDARDS.md#dot-variable-

**Comment by Loafin34** [comment]
Insane idea, what if you could dip burning items in beakers to add heat, like a welding tool? could be fun.

**Comment by ChungusGamer666** [comment]
I will be changing the value from 100º C to 150º C, as i realized that 100º C is quite pitiful for something that is burning (paper burns at around 218 celsius, for reference).

**Comment by msgerbs** [comment]
> Insane idea, what if you could dip burning items in beakers to add heat, like a welding tool? could be fun.

You can already use a welder on a beaker to heat it, can't you?

**Pull request merged**